content/docs/11.migration-guide/0.23.0/superadmin-refresh.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Manual user refresh to migrate Superadmin property
+icon: /docs/icons/migration-guide.svg
+release: 0.23.0
+editions: ["EE", "Cloud"]
+---
+
+## Overview
+
+The handling of Superadmin users in Kestra Cloud and Enterprise Edition has changed. Previously, the Superadmin status was determined by the user type (`SUPER_ADMIN`). In version 0.23, this is now managed through a dedicated property (`isSuperAdmin`). This change enables Service Accounts to have Superadmin permissions, but it also impacts user role detection for existing users.
```

Comment:
```suggestion
The handling of Superadmin users in Kestra Cloud and Enterprise Edition has changed. Previously, the Superadmin status was determined by the user type (`SUPER_ADMIN`). In version 0.23, this is now managed through a dedicated property (`isSuperAdmin`). This change enables new use cases such as assigning a Superadmin permission to a Service Account as well as sending an invite with Superadmin permissions, but it also impacts user role detection for existing users.
```

Reply:
@aj-emerich can you also edit the https://kestra.io/docs/enterprise/auth/rbac#super-admin please?

The new command to set/revoke superAdmin is the following:

`/kestra auths users set-superadmin user@email.com true` (`false` to revoke)